### ISSUE TITLE
Latest update enabling timeout, shown as unresponsive

### ISSUE THREAD

**Issue by ChromaPIE** [nobu-sh/vscode-mc-color#14]
Totally new to bug reports to vsc extensions so let me know if you need any log from somewhere.

**Comment by nobu-sh** [comment]
Yeah a log of sorts or a way to replicate it would be appreciated.

**Comment by nobu-sh** [comment]
Let me know if the latest version resolves the issue. I haven't been able to reproduce timeouts, even with 100k-line JavaScript files.

**Comment by ChromaPIE** [comment]
running v3.1.2, problem persists, shown as unresponsive. rolling back to v2 fixes the problem. please let me know where can i get the log, much appreciation!
